apps/web/examples/newPage/example.controller.js

```diff
@@ -0,0 +1,101 @@
+import logger from '../../src/server/lib/logger.js';
+import * as exampleService from './example.service.js';
+
+/** @typedef {import('./example.types').Example} Example */
+
+/**
+ * @typedef {object} ViewAllExamplesRenderOptions
+ * @property {Example[]} allExamplesFormatted
+ */
+
+/** @type {import('@pins/express').RenderHandler<ViewAllExamplesRenderOptions>}  */
+export async function viewAllExamples(request, response) {
+	const allExamples = await exampleService.getAllExample();
+
+	// Best to check that we have data, as we likely will not be able to display a page if we don't
+	if (allExamples) {
+		// Question: should we create mapper to contain the code below
+		const allExamplesFormatted = [];
+
+		for (const example of allExamples) {
```

Comment:
I would definitely introduce a formatter / factory here, depending of what extra processing the data needs before going to the template

Reply:
I agree! I will look into it.